### ISSUE TITLE
Error at process-classes phase

### ISSUE THREAD

**Issue by serdaryesilmurat** [vsilaev/tascalate-async-await#5]
I added to dependencies and plugins mentioned in readme. "mvn process-classes" gives the following error.

```
[ERROR] Cannot invoke "net.tascalate.async.tools.core.AsyncAwaitClassFileGenerator.reset()" because "generator" is null
java.lang.NullPointerException: Cannot invoke "net.tascalate.async.tools.core.AsyncAwaitClassFileGenerator.reset()" because "generator" is null
    at net.tascalate.async.tools.maven.AsyncAwaitEnhancerMojo.rewriteClassFile (AsyncAwaitEnhancerMojo.java:194)
    at net.tascalate.async.tools.maven.AsyncAwaitEnhancerMojo.execute (AsyncAwaitEnhancerMojo.java:156)
    at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo (DefaultBuildPluginManager.java:137)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:210)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:156)
    at org.apache.maven.lifecycle.internal.MojoExecutor.execute (MojoExecutor.java:148)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:117)
    at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject (LifecycleModuleBuilder.java:81)
    at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build (SingleThreadedBuilder.java:56)
    at org.apache.maven.lifecycle.internal.LifecycleStarter.execute (LifecycleStarter.java:128)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:305)
    at org.apache.maven.DefaultMaven.doExecute (DefaultMaven.java:192)
    at org.apache.maven.DefaultMaven.execute (DefaultMaven.java:105)
    at org.apache.maven.cli.MavenCli.execute (MavenCli.java:957)
    at org.apache.maven.cli.MavenCli.doMain (MavenCli.java:289)
    at org.apache.maven.cli.MavenCli.main (MavenCli.java:193)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0 (Native Method)
    at jdk.internal.reflect.NativeMethodAccessorImpl.invoke (NativeMethodAccessorImpl.java:77)
    at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke (DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke (Method.java:568)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced (Launcher.java:282)
    at org.codehaus.plexus.classworlds.launcher.Launcher.launch (Launcher.java:225)
    at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode (Launcher.java:406)
    at org.codehaus.plexus.classworlds.launcher.Launcher.main (Launcher.java:347)
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  20.270 s
[INFO] Finished at: 2022-01-12T10:52:09+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal net.tascalate.async:net.tascalate.async.tools.maven:1.1.0:tascalate-async-enhance (default) on project foo-service: Cannot invoke "net.tascalate.async.tools.core.AsyncAwaitClassFileGenerator.reset()" because "
generator" is null -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

```

Before "mvn process-classes", my code gives the following exception for an await(...) call at runtime:

> Continuation was continued incorrectly - are your classes instrumented for javaflow?

**Comment by vsilaev** [comment]
I don't know what is your pom.xml, what is your Maven version and what is your JDK version
Can't help without this. Try to use pom.xml from https://github.com/vsilaev/tascalate-async-await/tree/master/net.tascalate.async.examples as a starting point

**Comment by vsilaev** [comment]
fixed in release 1.2.0
